src/Security/Authorization/Core/src/AuthorizationServiceCollectionExtensions.cs

```diff
@@ -24,7 +24,11 @@ public static IServiceCollection AddAuthorizationCore(this IServiceCollection se
             {
                 throw new ArgumentNullException(nameof(services));
             }
-            
+
+            // These services depend on options, and they are used in Blazor WASM, where options
+            // aren't included by default.
+            services.AddOptions();
```

Comment:
Almost all of the AddXyz methods would potentially need this, but I guess the idea is to just add it to the one Blazor WASM needs for now?

Reply:
> Almost all of the AddXyz methods would potentially need this

I'm not 100% sure about that. 

99% of server-side ASP.NET's stuff won't work in Blazor WASM - and it doesn't really make sense for someone to try. If we really wanted to be thorough we could add this everywhere, but I'm not sure what the benefits really are.

This is the one thing I'm aware of that has usage in Blazor WASM.

Reply:
Logging and HttpClientFactory also add it, you're not using either of those?

Reply:
In Blazor WASM you're almost certainly not using those 😆

Reply:
This is a primary scenario and it's included in template, it's needs to be mistake-proof.